app/assets/stylesheets/darkswarm/cart-page.css.scss

```diff
@@ -0,0 +1,74 @@
+@import "mixins";
+@import "branding";
+@import "compass/css3/user-interface";
+@import "variables";
+
+#update-cart {
+  #errorExplanation {
+    display: none;
+  }
+}
+
+#cart-detail {
+  .cart-item-delete,
+  .bought-item-delete {
+    a {
+      font-size: 1.125em;
+    }
+  }
+
+  .out-of-stock {
+    color: $clr-brick;
+  }
+
+  button,
+  .button {
+    margin: 0;
+  }
+
+  .toggle-bought {
```

Comment:
have you considered using a `button` element instead? we wouldn't need this extra property then. It comes with it.

Reply:
Most of these bits are from moving or refactoring existing code, they're not actually changed at all. This rule is for an element in the cart page, which isn't touched in this PR at all...